pages/common/tldr.md

```diff
@@ -7,14 +7,14 @@
 
 `tldr {{command}}`
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Show the tldr page for `cd`, overriding the default platform(if the client supports -p):
 
 `tldr -p {{windows}} {{cd}}`
 
-- Show the tldr page for a subcommand:
+- Show the tldr page for a sub-command:
 
 `tldr {{git-checkout}}`
 
 - Update local pages (if the client supports caching):
 
-`tldr -u`
+`tldr --update`
```

Comment:
the -u is not included in the haskell platform either, only --update is, so its probably better to show the command for the more used platform

Reply:
This is not the first to make this change, but we can't do this because the long options are optional per the [client spec](https://github.com/tldr-pages/tldr/blob/ddf755f35520ab3b128255edd5da32a85f15bd02/CLIENT-SPECIFICATION.md#arguments):

>Clients MAY choose to only implement the short version of an option, ignoring the long form.